src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -330,7 +339,84 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.prefs.separatedPanel.showPanel = function (panel, callback) {
         // A bizarre race condition has emerged under FF where the iframe held within the panel does not
         // react synchronously to being shown
-        setTimeout(callback, 1);
+        fluid.invokeLater(callback);
+    };
+
+    fluid.defaults("fluid.prefs.separatedPanel.lazyLoad", {
```

Comment:
I think this grade could do with a big docs comment explaining its strategy - and outlining the expected sequence of activities - what happens first, which events are fired, and what they then trigger off. It should clearly contrast the standard fetch workflow of separatedPanel with the modified one and reference the JIRA and use cases for this functionality.